src/api.js

```diff
@@ -3,8 +3,12 @@ import axios from "axios";
 const news = axios.create({baseURL: 'https://northcoders-news-iumv.onrender.com/api/'})
 
 
-export const getArticles=((searchTerm) =>{
-    return news.get(`articles`, { params: { topic: searchTerm } })
+export const getArticles=((searchTerm,sort,order) =>{
+    return news.get(`articles`, { params: 
+        { topic: searchTerm,
+            sort_by: sort,
+            order: order} 
```

Comment:
for some reason when one of sort_by or order_by isn't selected then axios error is thrown.   have a look at this!